.github/workflows/ci_macos.yaml

```diff
@@ -1,6 +1,11 @@
 name: CI - macOS
+
 on: [pull_request, push]
 
+concurrency:
```

Comment:
Is it impossible to extract common parts in separate file and include it?

Reply:
I'm not aware of any possibility of it.